features/support/rummager_helpers.rb

```diff
@@ -0,0 +1,25 @@
+require "gds_api/test_helpers/rummager"
+
+module RummagerHelpers
+  include GdsApi::TestHelpers::Rummager
+
+  def stub_rummager
+    stub_any_rummager_post
+  end
+
+  def assert_policy_published_to_rummager(policy)
+    expected_json = JSON.parse({
```

Comment:
I followed the example in https://github.com/alphagov/contacts-admin/blob/aedb30231a3209c383d14503c798f2e21c6a6289/spec/models/contact_spec.rb#L14.